application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/api/QueryResult.java

```diff
@@ -0,0 +1,251 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha.api;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * See the Wolfram Alpha API.
+ */
+@JsonRootName("queryresult")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class QueryResult {
+    private static final XmlMapper XML = new XmlMapper();
+
+    @JacksonXmlProperty(isAttribute = true)
+    private boolean success;
+    @JsonIgnore
+    private boolean errorAttribute;
+    @JacksonXmlProperty(isAttribute = true, localName = "numpods")
+    private int numberOfPods;
+    @JacksonXmlProperty(isAttribute = true)
+    private String version;
+    @JacksonXmlProperty(isAttribute = true, localName = "datatypes")
+    private String dataTypes;
+    @JacksonXmlProperty(isAttribute = true)
+    private double timing;
+    @JacksonXmlProperty(isAttribute = true, localName = "timedout")
+    private String timedOutPods;
+    @JacksonXmlProperty(isAttribute = true, localName = "parsetiming")
+    private double parseTiming;
+    @JacksonXmlProperty(isAttribute = true, localName = "parsetimedout")
+    private boolean parseTimedOut;
+    @JacksonXmlProperty(isAttribute = true, localName = "recalculate")
+    private String recalculateUrl;
+
+    private Tips tips;
+    @JsonProperty("didyoumeans")
+    private DidYouMeans didYouMeans;
+    @JsonProperty("languagemsg")
+    private LanguageMessage languageMessage;
+    @JsonProperty("examplepage")
+    private ExamplePage examplePage;
+    @JsonProperty("futuretopic")
+    private FutureTopic futureTopic;
+    @JsonProperty("relatedexamples")
+    private RelatedExamples relatedExamples;
+    @JsonProperty("pod")
+    @JacksonXmlElementWrapper(useWrapping = false)
+    private List<Pod> pods;
+    @JsonIgnore
+    private org.togetherjava.tjbot.commands.mathcommands.wolframalpha.api.Error errorTag;
```

Comment:
I'm curious as to why this is FQN?

Reply:
Sorry, what is FQN?

Reply:
probably due to refactoring from the old code which had 2 classes. removed it, thanks.